src/api/integrations.js

```diff
@@ -51,6 +51,10 @@ IntegrationType.prototype.validate = function(props) {
 };
 
 const integrations = {
+    web: new IntegrationType([], ['brandColor', 'conversationColor', 'actionColor', 'displayStyle', 'businessName', 'businessIconUrl', 'buttonIconUrl', {
+        name: 'integrationOrder',
+        type: 'object'
```

Comment:
there's no `type: 'array'`?

Reply:
We're calling `typeof shizzle` and Arrays always return object 😭